ocaml/otherlibs/stdlib_alpha/capsule.mli

```diff
@@ -40,25 +40,68 @@
     this interface we will often use ['k] to refer to the capsule associated
     with that brand. *)
 
-(** Passwords represent access to a capsule. *)
+(** Converters can wrap and unwrap [Data.t] values from the current
+    capsule. *)
+module Converter : sig
+
+    (* CR layouts v5: this should have layout [void], but
+       [void] can't be used for function argument and return types yet. *)
+    type 'k t : value mod global portable many unique
```

Comment:
Why is this `portable`?

Reply:
As discussed offline, only because it safely can be not because it is actually useful. I prefer to reserve portability for controlling things that are like functions, and using contention for controlling things that are like mutable state, and I picture the converters as acting more like mutable state.